pkgs/top-level/all-packages.nix

```diff
@@ -13209,6 +13209,7 @@ in
   protobuf3_7 = callPackage ../development/libraries/protobuf/3.7.nix { };
   protobuf3_6 = callPackage ../development/libraries/protobuf/3.6.nix { };
   protobuf3_1 = callPackage ../development/libraries/protobuf/3.1.nix { };
+  protobuf3_0 = callPackage ../development/libraries/protobuf/3.0.nix { };
   protobuf2_5 = callPackage ../development/libraries/protobuf/2.5.nix { };
```

Comment:
Is this a protobuf thing that only very specific versions are compatible with things? If not it would be great if you could use 3.1 or later for this instead of adding another version

Reply:
Currently the package using an existing .deb that is compiled/packaged by Intel and linked against this specific protobuf version.

One solution to this would be to build the service from source, but from experience this may take longer to tweak, and it adds quite a bit of build time. (This is why I went for re-packaging the .deb)